sssd_test_framework/markers.py

```diff
@@ -67,7 +77,13 @@ def pytest_runtest_setup(item: pytest.Item) -> None:
             # @pytest.mark.builtwith(client="passkey", ipa="passkey") ->
             # -> check if "passkey" is supported by both client and ipa
             requirements = dict(mark.kwargs)
+        elif len(mark.args) == 1 and mark.args:
+            # @pytest.mark.builtwith(KnownTopology.IPA, ipa="passkey") ->
+            # -> check if "passkey" is supported by ipa
+            for iterargs in mark.args:
```

Comment:
```python
if isinstance(mark.args[0], Topology):
    topology = [mark.args[0]]
elif isinstance(mark.args[0], KnownTopology):
    topology = mark.arkgs[0].value
elif isintance(marks.args[0], KnownTopologyGroup):
    topology = [x.value for x in marks.args[0]]
```

Reply:
args of course. args are positional arguments (list), kwargs are keyword arguments (dictionary)

Reply:
I was asking because in your [previous comment](https://github.com/SSSD/sssd-test-framework/pull/48#discussion_r1334272951) you requested me to check `mark.kwargs` in the condition, but this variable isn't used anywhere.

Reply:
Disregard my last comment